server/src/main/kotlin/tech/figure/objectstore/gateway/configuration/DataConfig.kt

```diff
@@ -95,8 +100,23 @@ class DataConfig {
     fun flywayInitializer(flyway: Flyway): FlywayMigrationInitializer = FlywayMigrationInitializer(flyway)
 
     @Bean("MigrationsExecuted")
-    fun flywayMigration(dataSource: DataSource, databaseProperties: DatabaseProperties, flyway: Flyway): Int {
+    fun flywayMigration(
+        dataSource: DataSource,
+        databaseProperties: DatabaseProperties,
+        flyway: Flyway,
+        @Qualifier(BeanQualifiers.OBJECTSTORE_MASTER_KEY) masterKey: KeyRef,
+        provenanceProperties: ProvenanceProperties // todo: remove these two
+    ): Int {
         flyway.info().all().forEach { logger.info("Flyway migration: ${it.script}") }
-        return flyway.migrate().migrationsExecuted
+        return flyway.migrate().migrationsExecuted.also {
+            // todo: remove this block
+            val storageKeyAddress = masterKey.publicKey.getAddress(provenanceProperties.mainNet)
```

Comment:
didn't have a way to get the master key's address in a migration, so putting this here with the intent of removing it once we have done the migration where it matters (passport gateway instance is the only one that has any object permissions I believe)... this won't affect any records created after this migration, just the ones that have a null storage key address

Reply:
Might be good to be a little more elaborate in the todo message about which migration needs to run before this gets removed.  Since we publish each gateway version using semver, would be good to be explicit

Reply:
added a more elaborate message